### PR TITLE
trzsz-ssh 0.1.22

### PR DESCRIPTION
**Pull request by lonnywong**
Created with `brew bump-formula-pr`.<details>
  <summary>release notes</summary>
  <pre>## What's Changed
* expose tssh as a library
* tools: fix encode secret on Windows
* expect send text supports ssh tokens
* another way to support blocks feature in Warp
* support trzsz transfer in background
* support custom drag file upload command
* support custom progress color
* support clipboard integration via OSC52
* pass question to otp command
* fix tmux split panes #127
* tools: install tsshd to ~/.local/bin
* support pageant forwarding #131
* fix "early EOF" issue when using with git
* support custom udp port range
* support transferring files without dialog in command line
* feat: support --list-hosts tools command https://github.com/trzsz/trzsz-ssh/pull/134

## New Contributors
* @qcu266 made their first contribution in https://github.com/trzsz/trzsz-ssh/pull/134

**Full Changelog**: https://github.com/trzsz/trzsz-ssh/compare/v0.1.21...v0.1.22</pre>
</details>

### CODE REVIEW

**Comment by chenrui333** [comment]
```
  An exception occurred within a child process:
    Minitest::Assertion: Expected /trzsz\ ssh\ 0\.1\.22/ to match "".
```

File: Formula/t/trzsz-ssh.rb [hunk 1]
```diff
@@ -24,11 +24,11 @@ def install
...
-    assert_match "trzsz ssh #{version}", shell_output("#{bin}/tssh --version")
+    assert_match "trzsz ssh #{version}", shell_output("#{bin}/tssh -V 2>&1")
+    assert_match "trzsz ssh #{version}", shell_output("#{bin}/tssh --version 2>&1")
 
-    assert_match "invalid option", shell_output("#{bin}/tssh -o abc", 255)
-    assert_match "invalid bind specification", shell_output("#{bin}/tssh -D xyz", 255)
-    assert_match "invalid forward specification", shell_output("#{bin}/tssh -L 123", 255)
+    assert_match "invalid option", shell_output("#{bin}/tssh -o abc 2>&1", 255)
+    assert_match "invalid bind specification", shell_output("#{bin}/tssh -D xyz 2>&1", 255)
+    assert_match "invalid forward specification", shell_output("#{bin}/tssh -L 123 2>&1", 255)
```
Comment (chenrui333) [inline Formula/t/trzsz-ssh.rb:32]: but the thing is why do you output version info into stderr?

Comment (lonnywong) [reply Formula/t/trzsz-ssh.rb:32]: The original intention was to output the help message to stderr, but it affected the output of version.

https://github.com/trzsz/trzsz-ssh/blob/92d1967bdb24c0cab26428b49ed502c08d8bb9af/tssh/main.go#L122-L128

Comment (chenrui333) [reply Formula/t/trzsz-ssh.rb:32]: can we not output the version or help to the stderr? I dont think that is right practice?

Comment (lonnywong) [reply Formula/t/trzsz-ssh.rb:32]: But tssh v0.1.22 has been released and cannot be changed now.

Comment (chenrui333) [reply Formula/t/trzsz-ssh.rb:32]: (also you need to squash the commits to get rid of `autosquash` label and maybe good to add a note there too)

Comment (lonnywong) [reply Formula/t/trzsz-ssh.rb:32]: > can you log an issue and address in the future? thanks!

https://github.com/trzsz/trzsz-ssh/issues/136

**Review by carlocab — APPROVED** [review]

**Review by chenrui333 — APPROVED** [review]

**Pull request merged**